src/Compilers/CSharp/Test/Emit/Emit/OptionalArgumentsTests.cs

```diff
@@ -97,7 +97,10 @@ static void Main()
     }
     static void Report(object o)
     {
-        System.Console.WriteLine(""{0}: {1}"", o.GetType(), o);
+        var value = o is System.DateTime
+            ? ((System.DateTime)o).ToString(""MM/dd/yyyy HH:mm:ss"", System.Globalization.CultureInfo.GetCultureInfo(""en-US""))
```

Comment:
I don't **believe** the use of `GetCultureInfo` is necessary here. Part of the problem seems to be the use of `/` in the format string. These are not literal characters but instead locale specific date separators. If they are made to be actual literals using `'/'` then I believe this should pass without the need for using `GetCultureInfo`

Reply:
Tested this right now and it works fine using `'/'` and `':'`.
Calling `GetCultureInfo` might be more clear to anyone not knowing about the escaping rule. I don't have strong opinions of either so happy to change it if you think it's better to use the escaping.

Reply:
I'd prefer to using the escaping here if works. We try and only use specific cultures for formatting only when it's strictly required to get the test working.

Reply:
Changed it to use escaping now.